otx/cli/manager/config_manager.py

```diff
@@ -140,6 +146,8 @@ def configure_template(self, model: str = None) -> None:
         else:
             task_type = self.task_type
             if not task_type and not model:
+                if not hasattr(self.args, "train_data_roots"):
```

Comment:
This seems to be a duplicate of the Exception in auto_task_detection. Couldn't do without it?

Reply:
yes, when run "otx eval" without other argument, there will be AttributeError like below
![image](https://user-images.githubusercontent.com/5232387/226533604-8a9823d9-2b92-4b08-87c4-5e65e11b9ea9.png)